hackathon_site/hardware/api_urls.py

```diff
@@ -15,4 +15,5 @@
         name="hardware-detail",
     ),
     path("orders/<int:pk>/", views.OrderDetailView.as_view(), name="order-detail",),
+    path("order_items/", views.OrderItemListView.as_view(), name="order-detail",),
```

Comment:
should be called "order-list" instead of "order-detail"

Reply:
yep forgot, thanks